ykllvmwrap/src/jitmodbuilder.cc

```diff
@@ -610,9 +607,25 @@ class JITModBuilder {
     AllocaInst *ActiveFrameVec = FailBuilder.CreateAlloca(
         ActiveFrameSTy,
         ConstantInt::get(PointerSizedIntTy, ActiveFrames.size()));
+
+    std::vector<Value *> LiveValues;
     for (size_t I = 0; I < ActiveFrames.size(); I++) {
       FrameInfo FI = ActiveFrames[I];
 
+      // Read live AOT variables from stackmap calls and match them to
+      // variables in the JIT so we they can be optimised.
+      CallBase *SMC = FI.SMCall;
+      for (size_t Idx = 2; Idx < SMC->arg_size(); Idx++) {
```

Comment:
Should we define a constant for that `2`? Probably yes.

Reply:
I'd also be tempted to assert the instruction is the right kind (i.e. a stackmap call).

Reply:
Yeah, but still, code changes over time.

I don't feel mega-strongly, so I leave it up to you.

Reply:
Fixed in [3a22adb](https://github.com/ykjit/yk/pull/562/commits/3a22adbce65409477aa8f4e83c0b5e17f6da2045)